modules/imgur.py

```diff
@@ -1,14 +1,18 @@
 #--require-config imgur-api-key
 
-import re
-from src import ModuleManager, utils
+import re, json
+from src import ModuleManager, utils, EventManager
+from datetime import datetime
```

Comment:
please change to `import datetime` and further down use `datetime.datetime` - to keep in line with the rest of the project